src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main extends FigureSupplier {
+    private static final int NUMBER_OF_FIGURES = 6;
+    private static final int FIRST_AMOUNT = 3;
+
+    public static void main(String[] agr) {

```

Comment:
The parameter name 'agr' is likely a typo; it should be 'args' which is short for 'arguments'.